seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -809,7 +809,7 @@
         "assert.isTrue((/Purr\\s+jump\\s+eat/gi).test($(\"p\").text()), 'message: Your <code>p</code> element should contain the first few words of the provided additional <code>kitty ipsum text</code>.');",
         "assert($(\"p:eq(0)\").attr(\"class\") === \"red-text\", 'message: The first <code>p</code> element should have the class <code>red-text</code>.');",
         "assert($(\"p:eq(1)\").attr(\"class\") === undefined, 'message: Do not add a class attribute to the second <code>p</code> element.');",
-        "assert(parseInt($(\"p:not([class])\").css(\"font-size\"), 10) > 15, 'message: Between the <code>style</code> tags, give the <code>p</code> elements <code>font-size</code> of <code>16px</code>. Browser and Text zoom should be at 100%.');"
+        "assert(new RegExp(\"font-size: 16px;\", \"gi\").test(code), 'message: Between the <code>style</code> tags, give the <code>p</code> elements <code>font-size</code> of <code>16px</code>. Browser and Text zoom should be at 100%.');"
```

Comment:
The regEx for this test is both too rigid and not specific enough.

```html
<style>
h2 {
  font-size: 16px;
}
</style>
```
Would pass this test, and 
```html 
<style>
p {
  font-size:16px;
}
```
would fail the test.